src/app_charts/base/fluent-bit-helm.sh

```diff
@@ -10,6 +10,7 @@ sed -i '1i\{{ if and (eq .Values.robot_authentication "true") (eq .Values.fluent
 sed -i '1i\# !!! DO NOT EDIT THIS FILE !!!\n# This file is autogenerated using src/app_charts/base/fluent-bit-helm.sh.\n# See src/app_charts/base/README.md for update instructions.' ${OUTPUT}
 sed -i '$a\{{ end }}' ${OUTPUT}
 sed -i 's/MY_ROBOT/{{ .Values.robot.name }}/' ${OUTPUT}
+sed -i 's/Tag_Prefix kube.var.log.containers./Tag_Prefix {{ empty .Values.log_prefix_subdomain | ternary "" (list $.Values.log_prefix_subdomain "." | compact | join "") -}} kube.var.log.containers./' ${OUTPUT}
```

Comment:
This might deserve a comment.
`(list $.Values.log_prefix_subdomain "." | compact | join "")
is essentially appending the "." ?
`(print $.Values.log_prefix_subdomain ".")`
should also work

Reply:
Commented and corrected! https://github.com/googlecloudrobotics/core/pull/143/commits/15f35d09aab49711474058894152ea52a39bd22d